microsoft/testsuites/ltp/ltpsuite.py

```diff
@@ -19,6 +19,59 @@
 from lisa.tools import Lsblk, Swap
 from microsoft.testsuites.ltp.ltp import Ltp
 
+LTP_LITE_TESTS = ["math", "ipc", "mm", "sched", "pty", "fs"]
+LTP_REQUIRED_DISK_SIZE_IN_GB = 2
+
+
+class LtpTestConfig(NamedTuple):
+    test_list: List[str]
+    skip_test_list: List[str]
+    source_file: str
+    binary_file: str
+    git_tag: str
+    run_timeout: int
+    block_device: Optional[str]
+
+
+def get_ltp_config(node: Node, variables: Dict[str, Any]) -> LtpTestConfig:
+    test_str = variables.get("ltp_test", "")
+    skip_test_str = variables.get("ltp_skip_test", "")
+    source_file = variables.get("ltp_source_file", "")
+    git_tag = variables.get("ltp_tests_git_tag", "")
+    run_timeout = int(variables.get("ltp_run_timeout", 12000))
+    binary_file = variables.get("ltp_binary_file", "")
+    # block device is required for few ltp tests
+    # If not provided, we will find a disk with enough space
+    block_device = variables.get("ltp_block_device", None)
+
+    # get comma separated list of tests
+    if test_str:
+        test_list = test_str.split(",")
+    else:
+        test_list = LTP_LITE_TESTS  # fallback lite tests
```

Comment:
This is the only different code between lite and full test cases. All other logic should be shared. It doesn't need to create a test config class. Refer to smoke_test, how to use one method to serve multiple test cases.